### PR TITLE
Adding Build Workflow

### PR DESCRIPTION
**Pull request by 5jiji**
Adding a build workflow for automatically checking if the code is compilable & giving those builds back to anybody that whats to uses them

### CODE REVIEW

**Comment by zai-tm** [comment]
maybe

**Comment by RealMCoded** [comment]
possibly

File: .github/workflows/Build Windows.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# This workflow will build a .NET project
+# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-net
```
Comment (RealMCoded) [inline .github/workflows/Build Windows.yml]: no.

**Comment by 5jiji** [comment]
So uh... is it or not?

**Comment by zai-tm** [comment]
i actually have no idea what this does

**Comment by RealMCoded** [comment]
rootkit

**Comment by 5jiji** [comment]
This will, for both linux (ubuntu, latest version) & windows (server, because that's what github has, latest version): 
Install .NET
Resolve dependencies
Build
Test (if they exist)
Then upload the build as an artifact which will allow to download from github actions, (i have set the retention date to 30, so the artifact can be downloaded for up to 30 days)
Finish

**Comment by zai-tm** [comment]
sure

**Pull request merged**